src/main/java/design/MazeDesignerController.java

```diff
@@ -0,0 +1,40 @@
+package design;
+
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+
+
+public class MazeDesignerController {
+    private MazeDesignerInputBoundary md;
+
+    public MazeDesignerController(MazeDesignerInputBoundary md) {
+        this.md = md;
+    }
+
+    public void resetMaze(){
+        md.resetMaze();
+    }
+
+    public void handleBuild(int h, int row, int col){
```

Comment:
perhaps consider using a string instead of `int h`. 

for example, `String action`, and consider using a switch statement

Reply:
action can be one of "build", "remove", "start", or "end" or something of that sort